netsim/templates/provider/libvirt/cumulus_nvue-domain.j2

```diff
@@ -1,9 +1,10 @@
     $cumulus_script = <<-SCRIPT
 
 function set_cumulus_password(){
-    echo "### Setting cumulus user password to 'cumulus' ###"
+{% set _password = defaults.devices.cumulus_nvue.group_vars.ansible_ssh_pass %}
+    echo "### Setting cumulus user password to '{{ _password }}' ###"
     echo "cumulus ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/10_cumulus
-    echo "cumulus:GetLost1!" | chpasswd
+    echo "cumulus:{{ _password }}" | chpasswd
```

Comment:
FWIW, this works as long as nobody changes the ansible_ssh_pass variable for a node or a group. Better than hardcoded value though ;)

Reply:
Agreed, not perfect but better